FlowerShop/DTOs/CreateMerchandiseDTO.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FlowerShop.Models;
+using System.ComponentModel.DataAnnotations;
+
+namespace FlowerShop.DTOs
+{
+    public class CreateMerchandiseDTO 
+    {
+        [Required]
+        public string Name { get; set; }
+
+        [StringLength(maximumLength:255)]
+        public string? Description { get; set; }
```

Comment:
Gal geriau butu nurodyti StringLength modelyje o ne DTO

Reply:
Nope DTO geriau nes api is kart per request patikra, tai nereikia duombazej tikrint.